libraries/converter/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+#[cfg(feature = "arrow")]
+pub mod arrow;
+
+#[cfg(feature = "ndarray")]
+pub mod ndarray;
```

Comment:
would be great to support a kornia Image/Tensor converter 
- image: https://github.com/kornia/kornia-rs/tree/main/crates/kornia-image/src
- tensor: https://github.com/kornia/kornia-rs/tree/main/crates/kornia-image/src